src/BloomExe/Book/PublishSettings.cs

```diff
@@ -392,6 +392,31 @@ public EpubSettings Clone()
 			var serialized = JsonConvert.SerializeObject(this);
 			return JsonConvert.DeserializeObject<EpubSettings>(serialized);
 		}
+
+		public override bool Equals(object obj)
+		{
+			if (ReferenceEquals(null, obj)) return false;
```

Comment:
I was thinking that null EpubSettings should not be equal to anything, is that what we want? I'm also not sure if this line is necessary. Also on line 407